packages/plugin-core/src/test/suite-integ/NoteLookupCommand.test.ts

```diff
@@ -253,26 +267,168 @@ suite("NoteLookupCommand", function () {
           const out = (await cmd.run({
             noteType: LookupNoteTypeEnum.journal,
             noConfirm: true,
-          }));
+          })) as CommandOutput;
           
           expect(engine.config.journal.dateFormat).toEqual("y.MM.dd");
           // quickpick value should be `foo.journal.yyyy.mm.dd`
           const today = ((new Date()).toISOString().split("T")[0]).split("-").join(".");
```

Comment:
this will fail depending on local timezone. we generate journals based on local time but this is GMT time

Reply:
ah missed that. will fix.